.github/workflows/release.yaml

```diff
@@ -22,36 +22,35 @@ on:
   push:
     branches:
       - main
-      - "release/*"
+      - release/*
     paths:
-      - "versions.yaml"
+      - versions.yaml
   pull_request:
     branches:
       - main
-      - "release/*"
+      - release/*
     paths:
-      - "versions.yaml"
+      - versions.yaml
 
-permissions:
-  id-token: write
-  packages: write
-  pull-requests: write
-  contents: read
-
-env:
-  GITHUB_TOKEN: ${{ secrets.GH_RAD_CI_BOT_PAT }}
+permissions: {} # disable permissions
```

Comment:
nit: I think we should remove the `# disable permissions` comment or expand and say `# disable default GitHub token permissions`

Reply:
removed